view_acl/__init__.py

```diff
@@ -0,0 +1,15 @@
+from django.core import urlresolvers
+from django.views.defaults import permission_denied
+
+from .utils import get_caller_name
+
+def autodiscover():
+    """
+    Block all view defined in urls.py by default.
+    Return 403 status, permission denied.
+    Skipped '/admin' and etc
+    """
+    url_patterns = get_caller_name()
+    resolver = urlresolvers.get_resolver(url_patterns)
+    for url_pattern in resolver.url_patterns:
+        url_pattern._callback = permission_denied
```

Comment:
we don't have to force permission-denied for all url_patterns that we have. It's too risky.
Instead we need to have auth backend that authorizes user to have permissions for particular view. That backend module can be only responsible to permission_denied returning.